specifyweb/frontend/js_src/lib/components/Statistics/hooks.tsx

```diff
@@ -232,15 +181,24 @@ export function useResolvedStatSpec(
       };
     } else {
       const statSpecItem =
-        statsSpec[item.pageName][item.categoryName].items[item.itemName];
-      return item.itemType === 'BackEndStat'
+        statsSpec[item.pageName]?.categories?.[item.categoryName]?.items?.[
+          item.itemName
+        ];
+      return statSpecItem === undefined
+        ? undefined
+        : item.itemType === 'BackEndStat'
```

Comment:
Just replace `item.itemType === 'BackEndStat'` with `statSpecItem.spec.type === 'BackEndStat'` and then none of the type assertions are necessary